### DISCUSSION TITLE
Does Jupyter Book support include files? - getting error No Content permitted

### DISCUSSION THREAD

**Discussion by lwasser** [executablebooks/meta#824] [Q&A] [answered]
Hi There - i'm trying to include a chunk of text that I may want to use in a few spots in my book. i've tried a few things:

Classic Sphinx:

```.. include:: ../scope.md```

Using the directive feature

````
start it...
```{include} ../scope.md

```
end include
````

and i get:

```
ERROR: Directive 'include': No content permitted
```

I may not understand how the book infrastructure maps to regular sphinx here. is it possible to include a file?
Many thanks for any guidance.

**Comment by chrisjsewell** [comment, marked as answer]
Heya yeh just remove the blank line from the body of the include

**Reply by lwasser** [reply]
you rock -- that was the issue. it's all working now. I noticed that i needed to run `jupyter clean . ` and then `build .`  to get it to refresh properly. but it's working now!! thank you!!
